src/schema/schema_truncate.c

```diff
@@ -53,8 +53,11 @@ __truncate_tiered(WT_SESSION_IMPL *session, const char *uri, const char *cfg[])
     WT_STAT_DATA_INCR(session, cursor_truncate);
 
     /* Truncate the column groups. */
-    for (i = 0; i < tiered->ntiers; i++)
+    for (i = 0; i < WT_TIERED_MAX_TIERS; i++) {
+        if (tiered->tiers[i] == NULL)
+            continue;
```

Comment:
Will we have situations where a `NULL` entry in `tiered->tiers` is followed by other valid entries?  I.e., could we `break` rather than `continue` here?

Obviously with a table size of 4, this doesn't really matter from the performance perspective. I'm asking mostly to clarify the functionality we expect/need to support.

Reply:
Yes, I think we can have that situation, so `continue` is the right thing. We will have the situation (perhaps not in POC1) where we start only from the shared bucket. So a `tiered->tiers` would have the `tier:uri` at index 1 while it does not have a `file:uri` yet at index 0.

Reply:
Makes sense to me.  Thanks for clarifying.